tests/layer_tests/onnx_tests/test_lstm.py

```diff
@@ -140,6 +140,7 @@ def create_lstm(self, direction: str, cell_type: str, hidden_size=128):
         return onnx_net, None
 
     @pytest.mark.precommit
+    @pytest.mark.timeout(300)  # "Sporadic TimeoutError (Inference Engine running timed out)"
```

Comment:
This change won't fix the issue due inference has it's own timeout that should be increased for the model (see https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/common/layer_utils.py#L29)